egs/bn_music_speech/v1/conf/merge_vad.conf

```diff
@@ -0,0 +1,6 @@
+0 0 0
```

Comment:
I wouldn't call this merge_vad.conf '.conf' since it isn't the type of file that's read by the --config=xxx option to binaries.
If you could support comment lines in this, having a comment at the top of the file explaining what it means would be nice.